### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Microsoft.Identity.Client_ScreenReader:When we tab to the NuGet packages' table and GitHub repositories table the screen reader is not announcing the column headers name when its read the cell data

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9048]
### Impact

It's more difficult to complete my work

### Describe the bug

**Source:** https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468943

**Actual:**
When we tab to the NuGet packages' table and GitHub repositories table and move to each cell the column header of the respective cell is not read
Similar issue is also observed in JAWS
Similar issue is observed in throughout the application

**Expected:**
Each cell should be read with its name row column number and the column header name it belongs too.

**User Impact:**
Screen readers users will face difficulty in knowing the column header reference of the cell

### Repro Steps

1. Launch the application URL: https://www.nuget.org/packages/coverlet.msbuild/
2. "Microsoft.Identity.Client" page will get open.
3. Tab to 'Used by' tab controls and press enter to select it.
4. Tab to 'NuGet packages' table.
5. Run the NVDA
6. tab to the cells of the table   

### Expected Behavior

Each cell should be read with its name row column number and the column header name it belongs too.
